react/react-training/src/screens/MyScreenComponent/Board.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
```

Comment:
This is a component that belong to Game so create a folder component and move it and rename the file for `index.js`